modules/claims_api/lib/bd/bd.rb

```diff
@@ -80,20 +81,21 @@ def generate_upload_body(claim:, pdf_path:, file_number: nil)
     #
     # @return Faraday client
     def client
-      base_name = if !Settings.claims_api.evss_container&.auth_base_name.nil?
-                    "#{Settings.claims_api.evss_container.auth_base_name}/services"
+      base_name = if !Settings.claims_api.benefits_documents&.host.nil?
+                    "#{Settings.claims_api.benefits_documents.host}/services"
                   elsif @request&.host_with_port.nil?
                     'api.va.gov/services'
                   else
                     "#{@request&.host_with_port}/services"
                   end
 
       @token ||= ClaimsApi::V2::BenefitsDocuments::Service.new.get_auth_token
-      raise StandardError, 'Benefits Docs token missing' if @token.blank?
+      raise StandardError, 'Benefits Docs token missing' if @token.blank? && !@use_mock
```

Comment:
I would probably wrap these in an `unless @use_mock` instead of the `if`. Right now this gets a token, and then runs the check for `@use_mock`. if we're mocking, we have no reason to make this call. We'll just be necessarily adding to the volume of calls to retrieve a token.

Reply:
@FonzMP   I thought if use_mocks was set for this value it would not return a token https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/lib/claims_api/v2/benefits_documents/configuration.rb#L31